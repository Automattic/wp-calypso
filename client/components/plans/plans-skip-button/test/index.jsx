/**
 * @format
 */

/**
 * External dependencies
 */
import React from 'react';
import { shallow } from 'enzyme';

/**
 * Internal dependencies
 */
import { PlansSkipButton } from '../';

describe( 'PlansSkipButton', () => {
	test( 'should render', () => {
		const component = shallow( <PlansSkipButton /> );
		expect( component ).toMatchSnapshot();
	} );

	test( 'should render arrow-left in rtl mode', () => {
		const component = shallow( <PlansSkipButton isRtl /> );
		expect( component ).toMatchSnapshot();
	} );
} );
