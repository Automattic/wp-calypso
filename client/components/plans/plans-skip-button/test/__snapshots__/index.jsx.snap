// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlansSkipButton should render 1`] = `
<div
  className="plans-skip-button"
>
  <Button
    type="button"
  >
    Start with free
    <t
      icon="arrow-right"
      size={18}
    />
  </Button>
</div>
`;

exports[`PlansSkipButton should render arrow-left in rtl mode 1`] = `
<div
  className="plans-skip-button"
>
  <Button
    type="button"
  >
    Start with free
    <t
      icon="arrow-left"
      size={18}
    />
  </Button>
</div>
`;
